app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -323,14 +323,15 @@ private void addRoomColumns(HorizontalSnapScrollView scroller, List<Lecture> lec
         LinearLayout columnsLayout = (LinearLayout) scroller.getChildAt(0);
```

Comment:
:+1: Thank you for mentioning "and extract calculateDisplayDistance (duplicated code before)" in the commit message. It would have been hard to spot it in the diff otherwise.